swatch-contracts/ct/java/tests/ContractsComponentTest.java

```diff
@@ -297,4 +297,71 @@ private Subscription givenHypervisorSubscriptionIsCreated(String hypervisorSKU)
         is(HttpStatus.SC_OK));
     return hypervisorSubscription;
   }
+
+  @Test
+  @Tag("capacity")
+  void shouldValidateSumOfAllSocketsForPhysicalSkus() {
+    // Given: Get initial physical capacity via REST API
+    final OffsetDateTime beginning = OffsetDateTime.now().minusDays(1);
+    final OffsetDateTime ending = OffsetDateTime.now().plusDays(1);
+    final String physicalSku = RandomUtils.generateRandom();
+    double initialPhysicalSockets =
+        getPhysicalSocketCapacity(Product.RHEL, orgId, beginning, ending);
```

Comment:
Nitpick comment: this is not necessary because the org ID is auto-generated per each test, so the initial sockets will always be zero.